### PR TITLE
CFE-3990: Prompt user which bundle to run

### PR DESCRIPTION
**Pull request by larsewi**
**Demo:**
```
$ tree .
.
├── bogus.cf
├── cfbs.json
└── doofus
    ├── doofus.cf
    └── foo
        └── foo.cf

2 directories, 4 files
$ cfbs add ./bogus.cf 
Which bundle should be evaluated (added to bundle sequence)?
 1. ./bogus.cf:bogus (default)
 2. (None)
 [1/2]: 1
Added module: ./bogus.cf
$ cfbs add ./doofus/
Which bundle should be evaluated (added to bundle sequence)?
 1. ./doofus/doofus.cf:doofus (default)
 2. ./doofus/foo/foo.cf:foobar
 3. ./doofus/foo/foo.cf:foobaz
 4. (None)
 [1/2/3/4]: 2
Added module: ./doofus/
$ cfbs build
--snip--
002 ./bogus.cf  : copy './bogus.cf' 'masterfiles/services/cfbs/bogus.cf'
002 ./bogus.cf  : policy_files 'services/cfbs/bogus.cf'
002 ./bogus.cf  : bundles 'bogus'
003 ./doofus/   : directory './' 'masterfiles/services/cfbs/doofus/'
003 ./doofus/   : policy_files 'services/cfbs/doofus/doofus.cf' 'services/cfbs/doofus/foo/foo.cf'
003 ./doofus/   : bundles 'foobar'
--snip--
$ sudo cfbs install
Installed to /var/cfengine/masterfiles
$ sudo /var/cfengine/bin/cf-agent -KIf update.cf
--snip--
$ sudo /var/cfengine/bin/cf-agent -KI
R: This is /var/cfengine/inputs/services/cfbs/bogus.cf:bogus!
R: This is /var/cfengine/inputs/services/cfbs/doofus/foo/foo.cf:foobar!
```

### CODE REVIEW

**Review by olehermanse — APPROVED** [review]
Looks good, thanks for working on this!

File: cfbs/cfbs_config.py [hunk 1]
```diff
@@ -225,6 +227,79 @@ def _find_dependencies(self, modules, exclude):
             dependencies += self._find_dependencies(dependencies, exclude)
         return dependencies
 
+    def _add_to_inputs(self, module):
```
Comment (vpodzime) [inline cfbs/cfbs_config.py:208]: These are not inputs, but build steps.

File: cfbs/cfbs_config.py [hunk 2]
```diff
@@ -225,6 +227,79 @@ def _find_dependencies(self, modules, exclude):
...
+        name = module["name"]
+        step = "policy_files %s" % name
+        module["steps"].append(step)
+        log.debug("Added build step '%s' for module '%s'" % (step, name))
+
+    def _add_to_bundleseqence(self, module, policy_files):
```
Comment (vpodzime) [inline cfbs/cfbs_config.py:214]: *sequence*

File: cfbs/cfbs_config.py (hunk 1 above)
Comment (larsewi) [reply cfbs/cfbs_config.py:208]: See https://github.com/cfengine/cfbs/pull/134

**Pull request merged**